packages/fxa-auth-server/lib/senders/index.js

```diff
@@ -67,6 +67,7 @@ module.exports = async (
             ccEmails: cc,
             email: to || account.email,
             uid: account.uid,
+            metricsEnabled: !account.metricsOptOutAt,
```

Comment:
I can't say for certain that we pass an account record to all mailer functions (might be an object with email and uid). I think this is fine to do and then fallback to default of enabling metrics.

Reply:
Hmm, I wonder if that's why I ran into problems the first time I looked at this. I could've sworn I couldn't see the account data on at least one email, but on a second pass, it seemed to be present everywhere. I agree this seems fine either way, it's not worth the alternative😅